scripts/us_ca_chhs/healthcare_facility_bed_counts/preprocess_csv.py

```diff
@@ -0,0 +1,261 @@
+    'PEDI. DAY & RESPITE CARE': 'PediatricDayAndRespiteCareBed',
+    'DIALYSIS STATIONS': 'DialysisStationsBed',
+    'CORRECTIONAL TREATMENT CENTER': 'CorrectionalTreatmentCenterBed',
+}
+
+df2 = pd.read_excel(
+    'https://data.chhs.ca.gov/dataset/09b8ad0e-aca6-4147-b78d-bdaad872f30b/resource/d6599aac-ff5e-4693-a295-9f9d646a1f06/download/ca_county_gach_bed_counts.xlsx',
+    sheet_name='CA_COUNTY_GACH_BED_COUNTS')
+
+# Generate a dictionary from CA county name to geoId.
+counties = df2['COUNTY_NAME'].unique()
+county_to_geoID = generate_dcid_for_county(counties)
+
+elmsId_to_index = {}
+real_index = 0
+for index, row_dict in df1.iterrows():
+    elmsId = preserve_leading_zero(row_dict['FACID'])
+    stat_var = 'Count_HospitalBed_' + bed_type_dict[row_dict['BED_CAPACITY_TYPE']]
+    if elmsId in elmsId_to_index:
+        new_df.loc[elmsId_to_index[elmsId], stat_var] = row_dict['BED_CAPACITY']
+    else:
+        elmsId_to_index[elmsId] = real_index
+        real_index = real_index + 1
+        new_row = {
+            'GeoId': 'dcid:' + county_to_geoID[row_dict['COUNTY_NAME']],
+            'Date': '2020-06-01',
+            'CALicensedHealthcareFacilityBedFACID': 'ELMS/' + elmsId,
+            'CALicensedHealthcareFacilityName': row_dict['FACNAME'],
+            'CALicensedHealthcareFacilityType': 'dcs:' + FDR_to_type[row_dict['FAC_FDR']],
+            'Count_HospitalBed_SkilledNursingBed': None,
+            'Count_HospitalBed_SpecialTreatmentProgramBed': None,
+            'Count_HospitalBed_IntermediateCareHabilitativeBed': None,
+            'Count_HospitalBed_IntermediateCareNursingBed': None,
+            'Count_HospitalBed_CongregateLivingHealthFacilityBed': None,
+            'Count_HospitalBed_IntermediateCareBed': None,
+            'Count_HospitalBed_CoronaryCareBed': None,
+            'Count_HospitalBed_IntensiveCareBed': None,
+            'Count_HospitalBed_IntensiveCareNewbornNurseryBed': None,
+            'Count_HospitalBed_PerinatalBed': None,
+            'Count_HospitalBed_UnspecifiedGeneralAcuteCareBed': None,
+            'Count_HospitalBed_BurnBed': None,
+            'Count_HospitalBed_PediatricBed': None,
+            'Count_HospitalBed_RenalTransplantBed': None,
+            'Count_HospitalBed_RehabilitationBed': None,
+            'Count_HospitalBed_AcuteRespiratoryCareBed': None,
+            'Count_HospitalBed_AcutePsychiatricCareBed': None,
+            'Count_HospitalBed_ChemicalDependencyRecoveryBed': None,
+            'Count_HospitalBed_PediatricIntensiveCareUnitBed': None,
+            'Count_HospitalBed_LaborAndDeliveryBed': None,
+            'Count_HospitalBed_IntermediateCareDevelopmentallyDisabledBed': None,
+            'Count_HospitalBed_PsychiatricHealthBed': None,
+            'Count_HospitalBed_PediatricDayAndRespiteCareBed': None,
+            'Count_HospitalBed_DialysisStationsBed': None,
+            'Count_HospitalBed_CorrectionalTreatmentCenterBed': None
```

Comment:
Can you see if there is a way to add a row to a pd DataFrame without explicitly defining all values (default to None)?